uaclient/api/u/pro/token_info/v1.py

```diff
@@ -0,0 +1,237 @@
+import logging
+from typing import List
+
+from uaclient import exceptions, util
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.config import UAConfig
+from uaclient.contract import get_available_resources, get_contract_information
+from uaclient.data_types import (
+    BoolDataValue,
+    DataObject,
+    Field,
+    StringDataValue,
+    data_list,
+)
+from uaclient.entitlements import entitlement_factory
+from uaclient.status import _get_entitlement_information
+
+LOG = logging.getLogger(util.replace_top_level_logger_name(__name__))
+
+
+class TokenInfoOptions(DataObject):
+    fields = [
+        Field(
+            "token", StringDataValue, doc="Token to be used for the request"
+        ),
+    ]
+
+    def __init__(self, token: str):
+        self.token = token
+
+
+class accountInfo(DataObject):
```

Comment:
I think we need to rename the classes so that first letter is an `upcase`

Reply:
Updated @lucasmoura